demos/ripple.html

```diff
@@ -39,17 +39,17 @@
     <script src="assets/material-design-lite.js" charset="utf-8"></script>
     <script>
       (function(global) {
-        'use strict';
-        let surfaces = document.querySelectorAll('.surface');
-        for (let i = 0; i < surfaces.length; i++) {
-          let surface = surfaces[i];
-          let component = global.mdl.Ripple.attachTo(surface);
-          let button = surface.parentNode.querySelector('button');
-          button.addEventListener('click', () => {
-            let rect = surface.getBoundingClientRect();
+        var surfaces = document.querySelectorAll('.surface');
+        [].forEach.call(surfaces, function(surface) {
+          var rippleDom = global.mdl.Ripple.buildDom();
```

Comment:
It may be worth showing at least one example where the ripple DOM is constructed by the end user, rather than relying on `buildDom()`.